lib/index.js

```diff
@@ -92,14 +95,19 @@ const gatherMeta = async function gatherMeta(options, forcePath = true) {
 const createHandler = async function createHandler(options) {
   // this is where the actual create function is called and immediately
   // evaluated to determine if was successfully completed
+  const config = { ...options.config };
+  if (options.executionModel) {
+    config.executionModel = options.executionModel;
+  }
   const funcPath = path.resolve(options.path || process.cwd());
   await fse.mkdirp(funcPath);
   const funcName = await create(options.function, funcPath,
-    options.runtime, options.executionModel);
+    options.runtime, config);
   const optPath = options.path ? ` -p ${funcPath}` : '';
   const optName = options.function ? ` ${funcName}` : '';
+  const optConfig = verbose && config ? ` using ${inspect(config)}` : '';
   logger.info(
-`Created function ${funcName} in ${funcPath}
+`Created function ${funcName} in ${funcPath}${optConfig}
```

Comment:
Not sure I think the config should be printed (could go either way)

Reply:
it's not. `verbose` is `false`. look above. i started with it enabled and found it's too noisy, but decided to keep this extra "unused" code in case it's needed. if requested i'll happily delete it altogether.

Reply:
My bad, no problems.

Reply:
Although, sure why you wouldn't just `logger.verbose` with the config specifically?